libs/s25main/desktops/dskCredits.cpp

```diff
@@ -174,16 +290,31 @@ void dskCredits::Draw_()
     DrawCredit();
 }
 
+void dskCredits::GotoNextPage()
+{
+    // Next page
+    ++this->itCurEntry;
+    if(this->itCurEntry == entries.end())
+        this->itCurEntry = entries.begin();
+    timer.restart();
+}
+
+void dskCredits::GotoPrevPage()
+{
+    // Prev page
+    if(this->itCurEntry == entries.begin())
+        this->itCurEntry = std::prev(entries.end());
+    else
+        --this->itCurEntry;
+    timer.restart();
+}
+
 void dskCredits::DrawCredit()
 {
-    unsigned time = VIDEODRIVER.GetTickCount() - startTime;
+    const unsigned time = static_cast<unsigned>(std::chrono::duration_cast<std::chrono::milliseconds>(timer.getElapsed()).count());
```

Comment:
Just define `PAGE_TIME` as  a chrono type and don't convert at all

Reply:
doesnt work, you cant do an `a>b` comparison or similar with `std::chrono::milliseconds`?